app/views/api/v1/neighbors/search.json.jbuilder

```diff
@@ -0,0 +1,8 @@
+json.neighbors @neighbors do |neighbor|
```

Comment:
looks like this partial looks the same as above, cant you reuse only one?

Reply:
refactored